hedera-node/src/main/java/com/hedera/services/utils/EntityNumPair.java

```diff
@@ -92,4 +90,18 @@ public String toString() {
 				+ ", "
 				+ BitPackUtils.unsignedLowOrder32From(value) + ")";
 	}
+
+	@Override
+	public boolean equals(Object o) {
```

Comment:
What is the value of this method? We have a [unit test for `EntityNumPair.equals()`](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/test/java/com/hedera/services/utils/EntityNumPairTest.java#L49) that validates the expected semantics afaict... 🤔 

(If this `equals()` implementation is really necessary, that would be quite an alarming development!)